modules/apps/29-fee/keeper/escrow_test.go

```diff
@@ -140,142 +138,117 @@ func (suite *KeeperTestSuite) TestDistributeFee() {
 			// setup
 			refundAcc = suite.chainA.SenderAccount.GetAddress()
 			reverseRelayer = sdk.AccAddress(secp256k1.GenPrivKey().PubKey().Address())
-			forwardRelayer = sdk.AccAddress(secp256k1.GenPrivKey().PubKey().Address())
+			forwardRelayer = sdk.AccAddress(secp256k1.GenPrivKey().PubKey().Address()).String()
 
-			ackFee = validCoins
-			receiveFee = validCoins2
-			timeoutFee = validCoins3
-			packetId = &channeltypes.PacketId{ChannelId: suite.path.EndpointA.ChannelID, PortId: transfertypes.PortID, Sequence: validSeq}
-			fee := types.Fee{receiveFee, ackFee, timeoutFee}
+			packetId := &channeltypes.PacketId{ChannelId: suite.path.EndpointA.ChannelID, PortId: transfertypes.PortID, Sequence: validSeq}
+			fee := types.Fee{
+				ReceiveFee: defaultReceiveFee,
+				AckFee:     defaultAckFee,
+				TimeoutFee: defaultTimeoutFee,
+			}
 
 			// escrow the packet fee & store the fee in state
-			identifiedPacketFee := types.IdentifiedPacketFee{PacketId: packetId, Fee: fee, RefundAddress: refundAcc.String(), Relayers: []string{}}
+			identifiedPacketFee := types.IdentifiedPacketFee{
```

Comment:
nit: we have a [helper fn](https://github.com/cosmos/ibc-go/blob/ics29-fee-middleware/modules/apps/29-fee/types/msgs.go#L138) for this